ci/after-script.sh

```diff
@@ -15,6 +15,19 @@ handle_nodejs() {
   fi
 }
 
+handle_python() {
+  PYTHON_COV=services/"$SERVICE_TEST"/coverage/.coverage
+  COV_CONFIG=services/"$SERVICE_TEST"/setup.cfg
+
+  if [ -f "$PYTHON_COV" ]; then
+    sudo sed -i "s,/test/,$PWD/services/$SERVICE_TEST/,g" "$PYTHON_COV"
+    sudo sed -i "s,coverage/.coverage,$PYTHON_COV," "$COV_CONFIG"
+    pip install coveralls
```

Comment:
Why is sudo required for sed, but not for pip?

Reply:
I'm not sure, I think it has to do with the owner being set to root or something to the coverage files that are being outputted.